docs/src/main/sphinx/connector/druid.rst

```diff
@@ -57,6 +57,81 @@ name from the properties file.
 Type mapping
 ------------
 
+Because Trino and Druid each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Druid type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Druid types to the corresponding Trino types according to the
+following table:
+
+.. list-table:: Druid type to Trino type mapping
+  :widths: 30, 30, 50
+  :header-rows: 1
+
+  * - Druid type
+    - Trino type
```

Comment:
This looks reversed. `VARCHAR` is the Trino type, `STRING` is the Druid type.

Reply:
Done.